recipes/zstr/all/conanfile.py

```diff
@@ -0,0 +1,37 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class ZstrConan(ConanFile):
+    name = "zstr"
```

Comment:
I'm wondering whether we should namespace this recipe (`mateidavid-zstr`)

Reply:
I didn't find others libraries with this name: https://repology.org/projects/?search=zstr&maintainer=&category=&inrepo=&notinrepo=&repos=&families=&repos_newest=&families_newest=

Reply:
It's okay.
My suggestion would have applied to all (little) (header only) libraries.